### PR TITLE
add Activity

### PR DESCRIPTION
**Pull request by cediackermann**

### CODE REVIEW

File: Bash Scripting Language.md
Comment (TeamBattino) [inline Bash Scripting Language.md:1]: Nooooooo

**Review by TeamBattino — CHANGES_REQUESTED** [review]

**Review by TeamBattino — APPROVED** [review]
looks good

File: components/Sections/Activity/Activity.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import prisma, { Prisma } from "@/lib/prisma";
+import { WYSIWYG } from "@/components/Shared/WYSIWYGComponent";
+import style from "./Activities.module.css";
```
Comment (silvio2402) [inline components/Sections/Activity/Activity.tsx:4]: Here you import `./Activities.module.css`, but the actual file name is `./Activity.module.css`.

File: components/Sections/Activity/Activity.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    </div>
+  );
+}
+
+type ActivityBuilderProps = {
+  id: number;
+};
+
+export async function ActivityBuilder({ id }: ActivityBuilderProps) {
+  const activities = await prisma.activities.findFirst({
```
Comment (silvio2402) [inline components/Sections/Activity/Activity.tsx:33]: Rename this to `activity` since it's referring to one activity.

**Review by silvio2402 — CHANGES_REQUESTED** [review]

**Pull request merged**